### ISSUE TITLE
Battery icon disappearing from notification area

### ISSUE THREAD

**Issue by warpdesign** [microsoft/WinDev#57]
## Environment

| Item | Value | 
|---|---|
| OS, Version / Build | [version 10.0.19041.508] |
| Processor Architecture | x64 | 
| Processor Type & Model | core i5 | 
| Memory | 8 Gb |
| Storage Type, free / capacity (e.g. C: SSD 128GB / 512GB) | 256 Gb | 
| Relevant apps installed  | ___ |

## Description

Sometimes, the battery icon disappears from the notification area, even though the option is still enabled in the taskbar settings.

## Steps to reproduce

1.  Enable the battery icon in the taskbar settings

## Expected behavior

Battery icon should always be visible (either on the taskbar, or in the hidden notification icons that appear when clicking on the up arrow).

## Actual behavior

The battery icon isn't visible anymore.

Note: restarting the computer usually fixes the problem.

Feedback hub link: https://aka.ms/AAa0y2t

**Comment by LuanVSO** [comment]
restarting windows explorer from task manager also fix this.
btw this isn't really a dev issue 😅

**Comment by warpdesign** [comment]
@LuanVSO so you are having the same problem? Could you upvote the issue on GitHub and the feedback hub?

Thanks for the tip.

**Comment by Poopooracoocoo** [comment]
You can't upvote bugs on the Feedback Hub FYI. You used to be able to a year or two ago

**Comment by bitcrazed** [comment]
@warpdesign I assume you see this on a laptop/portable-device? Do you see the battery disappear when connected to mains power? On battery? Both? Does the battery disappear when fully charged?

**Comment by LuanVSO** [comment]
I leave mine on the charger, so generally the battery is full and connected at system bootup (after boot you either have or don't have the icon)

**Comment by orcmid** [comment]
> I leave mine on the charger, so generally the battery is full and connected at system bootup (after boot you either have or don't have the icon)

Side question: Do you know if the icon has moved off the system tray into the "show hidden icons" pop-up?  I have been having trouble getting things pinned to the tray and have to keep digging in there to shut off unwanted application remnants.

**Comment by LuanVSO** [comment]
>  Do you know if the icon has moved off the system tray into the "show hidden icons" pop-up?

no, the battery one is just gone. I've never had any problems with the hidden icons menu.

**Comment by warpdesign** [comment]
Same here: it's a Surface Book that's connected most of the time, but not always. The icon hasn't moved into the "hidden icons": it's just not there anymore.

The battery doesn't specifically disappear when fully charged (right now it's visible, and it's fully charged).

**Comment by bitcrazed** [comment]
Thanks for the info. Discussing with the Windows Power team. Will respond here when I hear back.

**Comment by warpdesign** [comment]
@bitcrazed I noticed another problem related to the battery icon that may be related. I submitted feedback+gh issue too: https://github.com/microsoft/WinDev/issues/61
